core/static/css/_main.scss

```diff
@@ -736,8 +736,10 @@ article {
     }
   }
 
-  .blurb-content {
+  .blurb{
     float: right;
+  }
+  .blurb-content {
```

Comment:
Can these styles just be moved to the individual class then? 
The blurb-content div was intended to be the wrapper doc originally, it seems like there is an extra layer in the loop now.